triggers/daml/Daml/Trigger.daml

```diff
@@ -260,3 +268,109 @@ data TriggerState s = TriggerState
   , commandsInFlight : Map CommandId [Command]
   , nextCommandId : Int
   }
+
+-- | Used to construct an 'ACS' for 'testRule'.
+newtype ACSBuilder = ACSBuilder [Update (AnyContractId, AnyTemplate)]
```

Comment:
`GeneralizedNewtypeDeriving` should allows us to write
```suggestion
newtype ACSBuilder = ACSBuilder [Update (AnyContractId, AnyTemplate)]
  deriving (Semigroup, Monoid)
```

Reply:
I tried that, but it fails with
```
Severity: DsError
Message:
  Failure to process DAML program, this feature is not currently supported.
  Coercion with tyconappco.
  ([Sym (N:ACSBuilder[0])])_R
```

Reply:
Openedhttps://github.com/digital-asset/daml/issues/4237 to track this

Reply:
Thanks!